moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -191,16 +191,17 @@ const ProjectSummaryKnackDataTrackerSync = ({
             project_id: project.project_id,
             knack_id: knackRecord.record.id,
           },
-        });
-      })
-      .then(() => refetch()) // ask the application to update its status from our graphql endpoint
-      .then(() => {
-        // End of the chain; advise the user of success
-        snackbarHandle(
-          true,
-          "Success: Project data synchronized with Data Tracker",
-          "success"
-        );
+        })
```

Comment:
Kind of hard to see in the diff, but these `then` calls were moved to chain from `mutateProjectKnackId` (the mutation to update the Moped project record) instead of the `fetch` that starts all of the updates